### PR TITLE
Matching Frontend-Backend Marriage

### PR DESCRIPTION
**Pull request by nickfong**
In lieu of doing two code reviews on frontend/backend of matching, consolidating into one PR.  Iff I can get this to pass CI before the backend PR is code reviewed, I'll close out #47 and #65.

### CODE REVIEW

File: features/step_definitions/matching.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Given(/^that the algorithm matches me with "(.*?)", "(.*?)", "(.*?)", "(.*?)", and "(.*?)"$/) do |match1, match2, match3, match4, match5|
+  pending
```
Comment (Fong-) [inline features/step_definitions/matching.rb]: @smartfuse can you tell me how I should force `matches` to return a preset list of users?

Comment (nickfong) [inline features/step_definitions/matching.rb:2]: @smartfuse can you tell me how I should force `matches` to return a preset list of users?

Comment (Fong-) [inline features/step_definitions/matching.rb]: @smartfuse can you tell me how I should force `matches` to return a preset list of users?

**Comment by nickfong** [comment]
Closing to base off of `iteration 4`

**Pull request closed without merging**